File: 백준/Bronze/2444. 별 찍기 － 7/별 찍기 － 7.py
# 마름모 변 입력받기
n = int(input())
N = 2 * n - 1
ans = 1
cnt = 1
while ans < n + 1:
    if cnt < n + 1:
        print(" " * (n - cnt) + "*" * (2 * cnt - 1))
        cnt += 1
    else:
        print(" " * ans + "*" * (2 * n - 2 * ans - 1))
        ans += 1

File: 백준/Bronze/2444. 별 찍기 － 7/별 찍기 － 7.js
const fs = require('fs');

// 입력을 파일로부터 읽어오는 방식
const input = fs.readFileSync('/dev/stdin').toString().trim();

// 입력을 숫자로 변환
const n = parseInt(input, 10);

// 첫 번째 피라미드 부분
for (let i = 1; i <= n; i++) {
    console.log(" ".repeat(n - i) + "*".repeat(i * 2 - 1));
}

// 두 번째 피라미드 부분
for (let j = 1; j < n; j++) {
    console.log(" ".repeat(j) + "*".repeat(n * 2 - j * 2 - 1));
}
